frontend/src/components/header/MobileLogo/MobileLogo.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {Typography} from "@mui/material";
+import './styles.css';
+
+const MobileLogo: React.FC = () => {
+    return (
+        <Typography
+            variant="h5"
+            noWrap
+            component="a"
+            className="mobile-logo mobile-logo-typography"
+        >
+            Qpay
```

Comment:
create folder src/constants
then files: app.ts
create constant APP_NAME and use it wherever you need. Same for titles, buttons and so on.